src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.cpp

```diff
@@ -208,9 +293,10 @@ bool ShenandoahGenerationSizer::adjust_generation_sizes() const {
     return false;
   }
 
-  if (_mmu_tracker->average() >= double(GCTimeRatio)) {
-    return false;
-  }
+  // With simplified MU implementation, we no longer use _mmu_tracker->average to trigger resizing
```

Comment:
Delete, rather than comment out. You can just leave a comment to motivate the deletion if you think that would help reviewers.